.github/workflows/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: PyInstaller Windows
+  uses: JackMcKew/pyinstaller-action-windows@main
+    with:
+      path: mathlibtools
```

Comment:
CI is failing on this file. I assume you never meant to commit it anyway?

Reply:
Apologies, the instructions said main.yml file is necessary, but I checked and it is not. Fine of course to rename the requirements file.

Reply:
Or I misread the instructions. I see the example he provided does not include a main.yml file.